plugin-tester-scala/src/main/scala/example/myapp/statefulhelloworld/GreeterServiceImpl.scala

```diff
@@ -0,0 +1,33 @@
+package example.myapp.statefulhelloworld
+
+import example.myapp.statefulhelloworld.grpc.GreeterService
+import example.myapp.statefulhelloworld.grpc.{ HelloRequest, HelloReply, ChangeRequest, ChangeResponse }
+import akka.actor.ActorRef
+import akka.actor.ActorSystem
+import akka.pattern.ask
+import akka.util.Timeout
+import scala.concurrent.duration._
+
+import scala.concurrent.Future
+
+// #stateful-service
+class GreeterServiceImpl(system: ActorSystem) extends GreeterService {
+
+  val greeterActor = system.actorOf(GreeterActor.props("Hello"), "greeter")
+
+  def sayHello(in: HelloRequest): Future[HelloReply] = {
+    // timeout and execution context for ask
+    implicit val timeout: Timeout = 3.seconds
+    import system.dispatcher
```

Comment:
I think we want to show `implicit val ec: ExecutionContext = system.dispatcher`?